src/set.js

```diff
@@ -108,5 +92,9 @@ export default class Set {
     return newArray
   }
 
-
+  forEach(element){
```

Comment:
`element` is a confusing name here - it should be a name that clearly identifies that this parameter is a callback.  (I like to use the identifier `callback` for this.  😄)